pydantic_ai_slim/pydantic_ai/providers/cerebras.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations as _annotations
+
+import os
+from typing import overload
+
+import httpx
+
+from pydantic_ai.exceptions import UserError
+from pydantic_ai.models import cached_async_http_client
+from pydantic_ai.profiles import ModelProfile
+from pydantic_ai.profiles.meta import meta_model_profile
+from pydantic_ai.profiles.openai import OpenAIJsonSchemaTransformer, OpenAIModelProfile, openai_model_profile
+from pydantic_ai.profiles.qwen import qwen_model_profile
+from pydantic_ai.providers import Provider
+
+try:
+    from openai import AsyncOpenAI
+except ImportError as _import_error:  # pragma: no cover
+    raise ImportError(
+        'Please install the `openai` package to use the Cerebras provider, '
+        'you can use the `openai` optional group — `pip install "pydantic-ai-slim[openai]"`'
+    ) from _import_error
+
+
+class CerebrasProvider(Provider[AsyncOpenAI]):
+    """Provider for Cerebras API."""
+
+    @property
+    def name(self) -> str:
+        return 'cerebras'
+
+    @property
+    def base_url(self) -> str:
+        return 'https://api.cerebras.ai/v1'
+
+    @property
+    def client(self) -> AsyncOpenAI:
+        return self._client
+
+    def model_profile(self, model_name: str) -> ModelProfile | None:
+        prefix_to_profile = {'llama': meta_model_profile, 'qwen': qwen_model_profile, 'gpt-oss': openai_model_profile}
+
+        profile = None
+        for prefix, profile_func in prefix_to_profile.items():
+            model_name = model_name.lower()
+            if model_name.startswith(prefix):
+                profile = profile_func(model_name)
+        else:
+            # TODO(Marcelo): We should create a profile for OpenAI Harmony: https://github.com/openai/harmony.
```

Comment:
Let's do this in this PR, as all the providers now support `gpt-oss` and should use the appropriate profile.

Reply:
Ah... I understand your comment now.

You assume the profile just needs what I wrote in the lines below, but I was expecting a lot more work than just moving that to a new profile.